afanasy/src/server/block.cpp

```diff
@@ -208,10 +208,10 @@ bool Block::v_startTask( af::TaskExec * taskexec, RenderAf * render, MonitorCont
    return true;
 }
 
-void Block::reconnectTask( af::TaskExec & i_taskexec, RenderAf & i_render, MonitorContainer * i_monitoring)
+void Block::reconnectTask(af::TaskExec *i_taskexec, RenderAf & i_render, MonitorContainer * i_monitoring)
 {
-	Task * task = m_tasks[i_taskexec.getTaskNum()];
-	task->reconnect( &i_taskexec, m_data->getRunningTasksCounter(), &i_render, i_monitoring);
```

Comment:
Note that I switched back the references to pointers here in arguments.
I prefer references in general, which is why I used it at first, but it does not make sense to "own" a reference as it makes for pointers, so it is cleaner imho to use pointers when there is a notion of owning transmission.